Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionViewController.swift

```diff
@@ -131,6 +132,25 @@ class ReviewExecutionViewController: ContainerViewController, PasscodeProtecting
     @IBAction func didTapPaymentMethod(_ sender: Any) {
         // open payment method selection  
         let choosePaymentVC = ChoosePaymentViewController()
+        choosePaymentVC.remainingRelays = controller.remainingRelays
+        choosePaymentVC.chooseRelay = {
```

Comment:
```suggestion
        choosePaymentVC.chooseRelay = { [unowned self] in
```

It should be like this, then you can remove usage of self from the call back